ansible/minigraph/switch4.xml

```diff
@@ -0,0 +1,916 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
@lguohan @qiluo-msft as far as I understand minigraph_facts use {{ inventory_hostname }} and use 'minigraph/{}.xml' to parse data i.e -mgmt ip. Deployment fails for me if I have switch4 in inventory but do not have corresponding xml under minigraph folder. 
Please correct me if I wrong.